variables.tf

```diff
@@ -0,0 +1,33 @@
+variable "vpc_name" {
+  description = "nome da vpc"
+  default     = ""
+}
+
+variable "project" {
+  description = "nome do projeto existente no google cloud"
+  default     = ""
+}
```

Comment:
Queremos em branco mesmo ou queremos forçar a pessoa a especificar?

Reply:
Decidimos remover o valor default para todos as variáveis que precisam ser especificadas.

Reply:
Colocar o type.